src/com/projectkorra/projectkorra/ability/EarthAbility.java

```diff
@@ -170,13 +170,14 @@ public boolean moveEarth(Block block, final Vector direction, final int chainlen
 							final LivingEntity lentity = (LivingEntity) entity;
 							if (lentity.getEyeLocation().getBlockX() == affectedblock.getX() && lentity.getEyeLocation().getBlockZ() == affectedblock.getZ()) {
 								if (!(entity instanceof FallingBlock)) {
-									entity.setVelocity(norm.clone().multiply(.75));
+									GeneralMethods.setVelocity(this,entity, norm.clone().multiply(.75));
+									
 								}
 							}
 						} else {
 							if (entity.getLocation().getBlockX() == affectedblock.getX() && entity.getLocation().getBlockZ() == affectedblock.getZ()) {
 								if (!(entity instanceof FallingBlock)) {
-									entity.setVelocity(norm.clone().multiply(.75));
+									GeneralMethods.setVelocity(this,entity, norm.clone().multiply(.75));
```

Comment:
Please change to `GeneralMethods.setVelocity(this, entity, norm.clone().multiply(.75));`